docs/docs/policies/rules.md

```diff
@@ -7,6 +7,62 @@ Rules determine which events a policy should trace.
 An event can match all occurrences of events for a specific scope, or specific events depending on its filters.
 Events support three types of filters: `context`, `arguments` and `return value`. 
 
+### Type of Events
+
+You can add as events either of the following:
+
+**[A syscall](../events/builtin/syscalls/index.md)**
+
+Example Scope Section referencing the `open` syscall:
+
+```bash
+spec:
+	scope:
+	    - global
+	rules:
+	    event: open
+```
+
+**[Network Events](../events/builtin/network.md)**
+
+Network Events can be specified from the list of `Available network events`.
+
+For example:
+
+```bash
+spec:
+	scope:
+	    - global
+	rules:
+	    event: net_packet_ipv4
+```
+
+**[A behavioural Signature](../events/builtin/signatures.md)**
+
+To specified one of the behavioral signatures as event, simply provide the name of the signature in the YAML manifest of the Tracee Policy:
+
+```bash
+spec:
+	scope:
+	    - global
+	rules:
+	    event: TRC-101
```

Comment:
```suggestion
	    event: TRC-101
```
TRCs are internal ids, we need to use the event name, soon TRCs won't be exposed anymore. 
eg: https://github.com/aquasecurity/tracee/blob/5d12c73031aaf8edc41552fdbbf091a71b6642fa/signatures/golang/proc_mem_access.go#L32

Reply:
What do you mean by ` soon TRCs won't be exposed anymore`?

Reply:
We will remove the TRCs ids from tracee.

Reply:
@AnaisUrlichs This one still needs to be fixed before we can merge this PR

Reply:
I changed it